lib/src/presentation/src/presentation_widget.dart

```diff
@@ -57,6 +58,21 @@ class PresentationWidget extends StatefulWidget {
   /// It will displayed when a slide doesn't have a background.
   final Widget? background;
 
+  /// An event callback when a slide is changed. The data passed in this
+  /// callback is [SlideChangeData]. The callback happens only when
+  /// a slide is changed. The index of slides start from 1.
```

Comment:
Isn't it contradicting? Index starts from 0 and here it's 1. Maybe we can change it to `slide position` or something?

Reply:
Not needed, index or position are almost same. Index can also start from 1.